calendar_event_security/security/calendar_event.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <record id="group_calendar_event_manager" model="res.groups">
+            <field name="name">Calendar: edit all events</field>
+            <field name="category_id" ref="base.module_category_usability"/>
+            <field name="users" eval="[(4, ref('base.user_admin'))]"/>
+        </record>
+    </data>
+    <data noupdate="1">
+        <record id="calendar.calendar_event_rule_employee" model="ir.rule">
+            <field ref="calendar.model_calendar_event" name="model_id"/>
+            <field name="name">Only Followers Can Write/Delete Calendar Events</field>
+            <field name="domain_force">[('partner_ids', 'in', [user.partner_id.id])]</field>
```

Comment:
I've realized that this contidition doesn't actually fit original requirements, that points to _Followers_, not Attendees. Please check